build.sbt

```diff
@@ -2,6 +2,12 @@ name := "root"
 
 Common.settings
 
+libraryDependencies ++= Seq(
```

Comment:
The I say move these into their respective projects. We can specify version numbers using the ```Common``` object to maintain consistency